inc/meta-box-html.php

```diff
@@ -459,8 +459,9 @@ protected function date( $taxonomy, $tax_nice_name, $multiples = false, $require
 					if ( is_numeric( $term->name ) ) {
 						$natdate = date( 'j F, Y', intval( $term->name ) );
 	?>
-			  <span><a id='<?php echo sanitize_text_field( $taxonomy ) ?>-check-num-<?php echo sanitize_text_field( $i ) ?>' class='ntdelbutton <?php echo sanitize_text_field( $term->name ) ?>'><?php echo sanitize_text_field( $term->name ) ?></a>&nbsp;<?php echo $natdate ?></span>
+			  <span><a id='<?php echo sanitize_text_field( $taxonomy ) ?>-check-num-<?php echo sanitize_text_field( $i ) ?>' class='datedelbutton <?php echo esc_attr( $term->name ) ?>'><?php echo sanitize_text_field( $term->name ) ?></a><?php echo $natdate ?></span>
 			<?php
+						HTML::hidden( 'rm_' . $tax_name . '_' . $i, null, null );
 					} else {
 						$date = strtotime( $term->name ); // If it isn't, convert it to a timestamp -- why? ?>
 			<span><a id='<?php echo sanitize_text_field( $taxonomy ) ?>-check-num-<?php echo sanitize_text_field( $i ) ?>' class='ntdelbutton <?php echo sanitize_text_field( $term->name ) ?>'><?php echo sanitize_text_field( $term->name ) ?></a>&nbsp;<?php echo sanitize_text_field( $term->name ) ?></span>
```

Comment:
And while you're in there, in the `class` attribute, change what it's echoing from

```php
<?php echo sanitize_text_field( $term->name ) ?>
```

to

```php
<?php echo sanitize_text_field( $date ) ?>
```

so we get the newly created timestamp, not a date string that includes spaces (which would mess with your `functions.js` line 80).